### DISCUSSION TITLE
Data summerize with group by

### DISCUSSION THREAD

**Discussion by JeroenP1974** [Budibase/budibase#13099] [Help] [answered]
Hi All, 

In a template there was a usage of views with 'group by' - function and option for calculate columns. This options is not availabe when I make a new view. Are these options not available anymore? Why?

Is there another way to aggregate my data and calculate columns? 

Thank you for the help.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @JeroenP1974,

See the explanation given here https://github.com/Budibase/budibase/discussions/12224#discussioncomment-7443835.

Hopefully this helps.

**Reply by JeroenP1974** [reply]
That is disappointing. Basic functions are simply not present. I'm now trying on a workaround. 

In a 'temp' table I have all kinds of columns with counts. Now I want to add the values ​​of these columns together in a screen or table. How do I do that? I don't see any columns with formulas anywhere.


<img width="1666" alt="Scherm­afbeelding 2024-02-22 om 13 12 22" src="https://github.com/Budibase/budibase/assets/157612513/c78bb861-de01-44de-abd1-403e238f12ab">


This is my table. And I want 2 cards in screen with : 
1) Grant total of column 'GroepVoorraadAantal'
2) Grant total of column 'GroepVoorraadWaarde' 

Both of these columns have Javascript for counting/grouping of data. 

Thank you for your help.
